src/Factory/Esabora/DossierMessageSCHSFactory.php

```diff
@@ -76,13 +76,17 @@ private function buildCommentaire(Signalement $signalement): string
     private function buildNbEnfants(Signalement $signalement)
     {
         $suffix = '';
-        if (str_ends_with($signalement->getNbEnfantsM6(), '+') || str_ends_with($signalement->getNbEnfantsP6(), '+')) {
+        if (null !== $signalement->getNbEnfantsM6() && str_ends_with($signalement->getNbEnfantsM6(), '+') ||
+            null !== $signalement->getNbEnfantsP6() && str_ends_with($signalement->getNbEnfantsP6(), '+')
+        ) {
             $suffix = '+';
         }
-        $nbEnfants = str_replace('+', '', $signalement->getNbEnfantsM6()) + str_replace('+', '', $signalement->getNbEnfantsP6());
+        $nbEnfantsM6 = (int) str_replace('+', '', $signalement->getNbEnfantsM6());
+        $nbEnfantsP6 = (int) str_replace('+', '', $signalement->getNbEnfantsP6());
+        $nbEnfants = $nbEnfantsM6 + $nbEnfantsP6;
```

Comment:
https://sentry.incubateur.net/organizations/betagouv/issues/73116/?query=is%3Aunresolved&referrer=issue-stream